build/mark-new-version.sh

```diff
@@ -24,7 +24,7 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 NEW_VERSION=${1-}
 
-VERSION_REGEX="v([0-9]+).([0-9]+(.[0-9]+)?)"
+VERSION_REGEX="v([0-9]+).([0-9]+)(.([0-9]+))?"
```

Comment:
No problem.  I was trying to make both work.  But i'm fine forcing the user to add .0